addons/web/static/src/core/popover/popover.js

```diff
@@ -171,6 +178,17 @@ export class Popover extends Component {
         }
     }
 
+    onHoverAway(ev) {
+        const target = ev.composedPath()[0];
+        if (!this.props.target.contains(target) && !this.popoverRef.el.contains(target)) {
+            this.closePopoverTimeout = browser.setTimeout(() => {
+                this.props.close();
+            }, 400);
+        } else {
+            browser.clearTimeout(this.closePopoverTimeout);
+        }
+    }
```

Comment:
The timeout should be a global concern, so that only one such popover can be hovered at a time.
Here by putting the responsibility on each popover, this is very prone to race conditions and buggy behaviour, e.g. you mouse-hover an item then quickly another one, and it shows both popovers or the 1st one (instead of the 2nd one).

By timers being a global concern, implementation is much easier and it's less prone to this buggy behaviour thanks to enforced singleton on presence of such "hover" popover.